misk/web/@misk/components/src/NavTopbarComponent.tsx

```diff
@@ -1,19 +1,83 @@
 import { Alignment, Navbar, NavbarDivider, NavbarGroup, NavbarHeading } from "@blueprintjs/core"
+import { IMiskAdminTabs } from "@misk/common"
 import * as React from "react"
 import { Link } from "react-router-dom"
+import styled from "styled-components"
 
 export interface ITopbarProps {
   name: string
   home: string
+  links: IMiskAdminTabs
 }
 
+const MiskNavbar = styled(Navbar)`
+  background-color: #29333a !important;
+  min-height: 74px;
+  margin-bottom: 20px;
+  box-sizing: border-box;
+  border: 1px solid #14191c;
+  padding-top: 10px;
+  position: fixed;
+  top: 0px;
+  
+  a {
+    color: #9da2a6;
+    text-decoration: none;
+  }
+  a:hover {
+    color: #fff;
+    text-decoration: none;
+  }
+`
+
+const MiskNavbarContent = styled.div`
+  margin: 0 auto;
+  width: 1000px;
+`
+
+const MiskNavbarGroup = styled(NavbarGroup)`
+  font-size: 13px !important;
+  font-weight: 600 !important;
+  line-height: 20px;
+  padding-left: 15px;
+  padding-right: 15px;
+  position: relative;
+`
+
+const MiskNavbarHeading = styled(NavbarHeading)`
+  font-size: 18px;
+  letter-spacing: 0px;
+  padding-top: 25px;
+  padding-bottom: 27px;
+`
+
+const MiskNavbarLink = styled(Link)`
+  font-size: 13px;
+  font-weight: 500;
+  text-transform: uppercase;
```

Comment:
I love this

Reply:
Really coming to love styled components, different way of thinking about building UI for web. 👍